liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -101,6 +101,9 @@
     @Parameter(property = "debugPort", defaultValue = "7777")
     private int libertyDebugPort;
 
+    @Parameter(property = "container", defaultValue = "false")
```

Comment:
Add 2020 to the copyright at the top of the file.

Reply:
done